backend/pycon/settings/base.py

```diff
@@ -202,3 +202,8 @@
 QUERY_INSPECT_TRACEBACK_ROOTS = [root(".")]
 
 PINPOINT_APPLICATION_ID = env("PINPOINT_APPLICATION_ID", default="")
+
+
+MAILCHIMP_SECRET_KEY = env("MAILCHIMP_SECRET_KEY")
+MAILCHIMP_DC = "us5"
+MAILCHIMP_LIST_ID = env("MAILCHIMP_LIST_ID")
```

Comment:
I think we can make those optional and in the `subscribe` of mailchimp if `MAILCHIMP_SECRET_KEY` is None raise an error saying `Mailchimp integration is not enabled`

What do you think?

Reply:
raise something directly in the settings file? or only in prod file?